flink-clients/src/main/java/org/apache/flink/client/cli/CliFrontend.java

```diff
@@ -1001,9 +1009,10 @@ public int parseParameters(String[] args) {
 			return handleParametrizationException(ppe);
 		} catch (ProgramMissingJobException pmje) {
 			return handleMissingJobException();
-		} catch (Exception e) {
-			return handleError(e);
 		}
+//		catch (Exception e) {
```

Comment:
Shouldn't this be either removed or uncommented?

Reply:
Yes, this is the leftover I mentioned in my first comment. I've reworked the test to instead check stdout printing.